### ISSUE TITLE
Close options of the menu are awkward

### ISSUE THREAD

**Issue by dlgski** [angular-ui/ui-grid#5076]
When you open a menu, there is a close X that gains focus if you tab into the menu or when using a screen reader.

Also, when tabbing through the menu options, a hidden Close menu option shows up on tab only.  If the grid has any custom menu options (grouping, aggregation) this close option is injected right in the middle of the menu.

Desired behavior:  Screenreaders and power-users expect that an ESC key will close popups of all kids.  There is no need for the X or the Close option, but ESC should be implemented to close the menu.
